spark/src/main/scala/org/apache/spark/sql/delta/DeltaParquetFileFormat.scala

```diff
@@ -266,7 +268,11 @@ case class DeltaParquetFileFormat(
     val factory = super.prepareWrite(sparkSession, job, options, dataSchema)
     val conf = ContextUtil.getConfiguration(job)
     // If the key is not set explicitly, default to INT64 for Iceberg compatability
-    if (!sparkSession.conf.contains(SQLConf.PARQUET_OUTPUT_TIMESTAMP_TYPE.key)) {
+    val parquetOutputTimestampType = sparkSession.conf.get(
```

Comment:
uhm i am not sure how this would work... 
https://github.com/apache/spark/blob/5d16c3134c442a5546251fd7c42b1da9fdf3969e/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L1056

iiuc. the spark conf should already gives a default, so idk if here it will return int96 or undefined... 

I think maybe a cleaner way is to duplicate PARQUET_OUTPUT_TIMESTAMP_TYPE to DeltaSQLConf, with default of 64, and only use that in this DeltaParquetFileFormat ?

Reply:
Thanks @lzlfred  for the suggestion, and I like that very much. I've updated the PR accordingly 👍